tests/benchdnn/utils/compare.cpp

```diff
@@ -267,9 +267,11 @@ int compare_t::compare_p2p(const dnn_mem_t &exp_mem, const dnn_mem_t &got_mem,
 
     dnn_mem_t got_f32(got_mem, dnnl_f32, tag::abx, get_cpu_engine());
     dnn_mem_t exp_f32_plain;
-    if (has_prim_ref_
-            && (query_md_data_type(exp_mem.md_) != dnnl_f32
-                    || !check_md_consistency_with_tag(exp_mem.md_, tag::abx))) {
+    const bool is_complaint_prim_ref_dst_mem
```

Comment:
complaint?

Reply:
It's a personal version of "compliant". I'll change, thanks.